kitipy/context.py

```diff
@@ -68,9 +68,13 @@ def __init__(self,
         self.config = config
         self.stage = stage
         self.stack = stack
-        self.executor = executor
+        self._executor = executor
```

Comment:
Don't you have to call `super()` here? The `_executor` is already setted in `ProxyExecutor:__init__()`

Reply:
Yeah indeed, it's a remainder of a previous implementation.